lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -1069,7 +1074,21 @@ void HdVP2Material::_ApplyVP2Fixes(HdMaterialNetwork& outNet, const HdMaterialNe
         }
 
         addPredecessorNodes(node);
-        outNet.nodes.push_back(node);
+        if (_IsUsdUVTexture(node)) {
```

Comment:
I think this code fits better with the code around line 880. That code is already renaming some other nodes, so renaming the texture nodes fits better there then it does here.